plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/conversions/NullabilityConversion.kt

```diff
@@ -33,4 +39,19 @@ class NullabilityConversion(context: NewJ2kConverterContext) : RecursiveConversi
             type.type = type.type.updateNullability(NotNull)
         }
     }
-}
+    private fun JKMethod.updateNullabilityOfParameters() {
+        val enumConstants = mutableListOf<JKEnumConstant>()
+        parentOfType<JKClassBody>()?.forEachDescendantOfType<JKEnumConstant> {
+            enumConstants.add(it)
+        }
+        for (i in parameters.indices) {
+            val isNotNull = enumConstants.none {
+                val argument = it.arguments.arguments.getOrNull(i)
+                argument == null || argument.value !is JKLiteralExpression
```

Comment:
It's better to directly check the nullability of the expression's type and not hardcode `JKLiteralExpression`.